dashboard/.streamlit/config.toml

```diff
@@ -0,0 +1,10 @@
+[browser]
+gatherUsageStats = false
+
+[theme]
+base = "light"
+primaryColor = "#3c9aaa"
+
```

Comment:
Don't see anything wrong with this, would've preferred dark mode but personal preference